### PR TITLE
b/152394779: put gcsrunner in official release images

### PR DESCRIPTION
**Pull request by TAOXUY**

### CODE REVIEW

File: scripts/all-utilities.sh [hunk 1]
```diff
@@ -310,11 +310,15 @@ function get_serverless_image_name() {
...
+  echo -n "gcr.io/${ESP_RELEASE_PROJECT}/endpoints-runtime"
 }
 
 function get_serverless_image_release_name() {
-  echo -n 'gcr.io/endpoints-release/endpoints-runtime-serverless'
+  echo -n "gcr.io/${ESP_RELEASE_PROJECT}/endpoints-runtime-serverless"
+}
+
+function get_gcsrunner_image_release_name() {
+  echo -n "gcr.io/${ESP_RELEASE_PROJECT}/gcsrunner"
```
Comment (qiwzhang) [inline scripts/all-utilities.sh]: can we keep it in our cloudese-testing project for gcsrunner?

endpoints-release is for users.  gcsrunner is not for users.

Comment (KyleWiese) [reply scripts/all-utilities.sh]: Xuyang made a good comment offline that `cloudesf-testing` would also have all the CI builds in it which might get messy. If we don't want to put it in `endpoints-release`, would it be preferable for us to have a separate GCR repo for the gcsrunner releases?

Comment (TAOXUY) [reply scripts/all-utilities.sh]: Yes, we can do that. The destination project totally depends on you guys.

Comment (KyleWiese) [reply scripts/all-utilities.sh]: Okay, let me talk with the team to see what we want it to be named.

Comment (TAOXUY) [reply scripts/all-utilities.sh]: Updated to be ` gcr.io/espv2-gcsrunner-release/gcsrunner`

PTAL

File: scripts/release/release-stable.sh [hunk 2]
```diff
@@ -29,6 +29,8 @@ This script will release stable ESPv2 docker image with format of:
   $(get_proxy_image_release_name):\${MAJOR_BASE_VERSION}
   $(get_serverless_image_release_name):\${MINOR_BASE_VERSION}
   $(get_serverless_image_release_name):\${MAJOR_BASE_VERSION}
+  $(get_gcsrunner_image_release_name):\${MINOR_BASE_VERSION}
+  $(get_gcsrruner_image_release_name):\${MAJOR_BASE_VERSION}
```
Comment (KyleWiese) [inline scripts/release/release-stable.sh]: Is this where the patch version would go? Or does that part of the tagging happen separately?

Comment (TAOXUY) [reply scripts/release/release-stable.sh]: We haven't put the patch part. It is only the release part.

Comment (KyleWiese) [reply scripts/release/release-stable.sh]: Gotcha. LGTM

**Comment by TAOXUY** [comment]
Change only involves release scripts so skip the CI.

**Pull request merged**